instrumentation/rails/test/test_helpers/configs/rails6.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module Rails6
+  class Application < Rails::Application
+    config.eager_load = false # Ensure we don't see this Rails warning when testing
+    config.logger = Logger.new('/dev/null') # Prevent tests from creating log/*.log
+    config.hosts << 'example.org'
+    config.secret_key_base = 'secret_key_base'
+  end
+end
+
+def build_app(use_exceptions_controller: false)
+  require 'action_controller/railtie'
+  require 'test_helpers/middlewares'
+
+  new_app = ::Rails6::Application.new
+  new_app.config.secret_key_base = 'secret_key_base'
+
+  # Ensure we don't see this Rails warning when testing
+  new_app.config.eager_load = false
+
+  # Prevent tests from creating log/*.log
+  new_app.config.logger = Logger.new('/dev/null')
+
+  # Required in Rails 6
+  new_app.config.hosts << 'example.org'
```

Comment:
Same question here - do you need to set these in both places?

Reply:
The above config was not needed as the `.new` call doesn't carry over any of that config.  I've removed that duplication. In both places.